### PR TITLE
Update Ubuntu VM Image from 18 to 20

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update Ubuntu VM image from 18 to 20

### CODE REVIEW

File: eng/pipelines/templates/jobs/archetype-sdk-client.yml [hunk 1]
```diff
@@ -27,8 +27,8 @@ jobs:
   strategy:
     matrix:
       Linux_x64_gcc8:
-        Pool: azsdk-pool-mms-ubuntu-1804-general
-        OSVmImage: MMSUbuntu18.04
+        Pool: azsdk-pool-mms-ubuntu-2004-general
+        OSVmImage: MMSUbuntu20.04
         VcpkgInstall: 'curl[ssl] libxml2 openssl'
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/archetype-sdk-client.yml:32]: What about the other ubuntu entries below?

**Comment by chidozieononiwu** [comment]
@RickWinter Can I go ahead and merge this?

**Comment by ahsonkhan** [comment]
> @RickWinter Rick Winter FTE Can I go ahead and merge this?

Please hold off for a couple of days if possible, as we work through a release, to avoid destabilizing main.

**Comment by weshaggard** [comment]
@RickWinter @ahsonkhan is this good for merging now the release is complete?

**Review by gearama — APPROVED** [review]

**Pull request merged**